### PR TITLE
Update client.dart

### PR DESCRIPTION
**Pull request by andromeda911**
Web3Client class have method signTransaction(), but we can`t send raw signed transaction using public API of this lib.

### CODE REVIEW

File: lib/src/core/client.dart [hunk 1]
```diff
@@ -279,7 +279,18 @@ class Web3Client {
     return _makeRPCCall('eth_sendRawTransaction',
         [bytesToHex(signed, include0x: true, padToEvenLength: true)]);
```
Comment (simolus3) [inline lib/src/core/client.dart:280]: Thanks for your contribution. It looks good, but let's change `sendTransaction` to also use the new method:
```suggestion
    return sendRawTransaction(signed);
```

**Comment by andromeda911** [comment]
Why build fails?

**Comment by simolus3** [comment]
It looks like the analyzer is complaining:

```
Analyzing web3dart...
  lint • Only reference in scope identifiers in doc comments. • lib/src/core/client.dart:282:63 • comment_references
1 lint found.
```

I fixed that, it should hopefully work now.

**Comment by simolus3** [comment]
Released this as 2.0.0-dev.9. Thanks for your contribution!

**Pull request merged**